src/bun.js/api/bun.zig

```diff
@@ -486,6 +486,18 @@ pub fn getImportedStyles(
     return JSValue.createStringArray(ctx.ptr(), styles.ptr, styles.len, true).asRef();
 }
 
+pub fn newOs(
```

Comment:
There's a new and better way to expose ESM modules, but most of the code hasn't been migrated to it. Have a look at ProcessModule.h. 

Instead of adding a new property and writing JS wrapper code, we can directly expose the JSValue objects